### ISSUE TITLE
Searchlight type flare launcher

### ISSUE THREAD

**Issue by BLOODWOLF333** [MegaMek/megamek#201]
i know this isn't canon but, we have this tech now days and I would imagine this would be nothing during the Battletech age.  I think it would be useful and make night fighting less difficult if mechs and vehicles could have a flare launcher tacked onto them like how it is done with searchlights.  Maybe have only 1-4 shots and yes it may make that mech flare mortar obsolete but, who would want to waste tonnage on a flare launcher when now days infantry we can carry pop-up flares on our person to light up the battlefield during night ops?  Obviously I abhor the cannon night TH modifiers considering mechs have night vision and thermal optics but, I understand those are the canon rules.

**Comment by BLOODWOLF333** [comment]
flare.png
![flare](https://cloud.githubusercontent.com/assets/16950832/15659461/4ccd6b90-2690-11e6-8bad-5c33b0ae0bac.png)

**Comment by pheonixstorm** [comment]
Even night vision and thermal has its limits in range.

**Comment by BLOODWOLF333** [comment]
Right, our NVGs in the military wouldnt work in a pitch black scenario since they need at least starlight ambient light.  So, in my head I think it would be neat for a defenders of a base you are attacking at night to have search lights and flare launchers that they try to illuminate your forces as you sneak up.  Then you would have to launch smoke grenades to conceal your forces in return etc.

**Comment by BLOODWOLF333** [comment]
Half of my dream came true with @hammer's turrets now able to use flares and smoke rounds for mortars so thats going to be a neat feature.
